Addressing Lars' comments (#20)
lamps-wg/documentsigning-eku: draft-ietf-lamps-documentsigning-eku.md

# Using the Extended Key Purpose for Document Signing in a Certificate

> The signed contents of documents are primarily intended to be
> consumed by people. To be more precise, contents are intended to be

Author: ```suggestion
Our intended use case is people consuming the contents of signed
documents. To be more precise, contents are intended to be
```

When a single application has the capability to process various data
formats, the software may choose to make the excluded and permitted
decisions separately in accordance with the format it is handling (e.g.,

> TEXT, PDF).

Reviewer: I think text could stay lowercase.
